pages/bookmarks.jsx

```diff
@@ -1,16 +1,23 @@
 import React from 'react';
```

Comment:
hey @Pomax can I suggest that we open a followup ticket for this? I would like to have this already-huge PR merged first if possible. I'll get to the followup ticket right after this PR is merged.